pkg/instrumentation/sdk_test.go

```diff
@@ -1101,6 +1101,141 @@ func TestInjectGo(t *testing.T) {
 	}
 }
 
+func TestInjectApacheHttpd(t *testing.T) {
```

Comment:
nit: can you restructure this test to follow the same "name, inst, pod, expected" struct pattern that the other tests follow?

Reply:
Restructured with the test struct as advised, thank you - much more readable and scalable.